bot/resources/valentines/zodiac_explanation.json

```
@@ -0,0 +1,122 @@
+{
+    "Aries": {
+            "start_at":[3, 21],
+            "end_at": [4, 19],
```

Using dates with ISO format will be much more readable in my opinion. So, you could change it to:
```suggestion
            "start_at": "2020-03-21",
            "end_at": "2020-04-19",
```
This has to be done to all zodiac signs in the file.